iam-openidconnectprovider/cmd/resource/model.go

```diff
@@ -1,7 +1,5 @@
 package resource
 
-import "github.com/aws-cloudformation/cloudformation-cli-go-plugin/cfn/encoding"
-
 /*
 This file is autogenerated, do not edit;
```

Comment:
Why do we have this in VC?

Reply:
This file is an essential part of the project and shall be available even without `cfn generate` run. Without this file, in version control, upstream changes won't be visible.